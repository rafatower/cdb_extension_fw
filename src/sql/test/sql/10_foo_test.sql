SELECT cdb_extension_fw.foo();
