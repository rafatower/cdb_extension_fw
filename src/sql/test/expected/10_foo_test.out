SELECT cdb_extension_fw.foo();
NOTICE:  foo
 foo 
-----
 
(1 row)

